src/controllers/graphql/resolvers/user/userResolver.js

```diff
@@ -1,11 +1,21 @@
+const { AuthenticationError } = require('apollo-server');
 const db = require('../../connectors');
-
 const User = db.sequelize.models.user;
 
 module.exports = {
   Query: {
     user(_, args) {
       return User.find({ where: args });
+    },
+    follow: async (_, args, { authenticatedUserInstance }) => {
```

Comment:
`follow` and `unfollow` change the state of the user, so they should be categorized under `Mutations`. Queries are reserved for fetching information

Reply:
Ok sure

Reply:
followers and unfollowers are not stored in our server db, so they are in fact a query.  Yes they are changing the user instance on the blockchain, but not serverside.